poc/pkg/values.yaml

```diff
@@ -67,19 +67,24 @@ tolerations: []
 affinity: {}
 
 # Use for Istio service mesh 
-#TODO jonachang: need to add a secret in other PR
 istio:
   name: aks-istio-ingressgateway-external
   gateway:
     name: rp-poc-gateway
-    #TODO jonachang: change back to 443 when we have TLS
-    port: 80
+    port: 443
+    # TLS Mode: https://istio.io/latest/docs/reference/config/networking/gateway/#ServerTLSSettings-TLSmode
+    tlsMode: SIMPLE
   virtualService:
     name: rp-poc-virtualservice
+  tlsDeployment:
+    name: tls-sync-deployment
+    image: mcr.microsoft.com/azuredocs/aks-helloworld:v1
+    container: tls-sync-container
+    port: 80
```

Comment:
Does `aks-helloworld` require a port to be open? Ideally, we don't open a port given the purpose of this pod is to sync a secret.